content/v2.1/en/introduction/what-is-kubesphere.md

```diff
@@ -6,11 +6,11 @@ description: 'What is KubeSphere'
+**KubeSphere** is a multi-tenant enterprise-grade container platform built on [Kubernetes](http://kubernetes.io), with full-stack automated IT operation and streamlined DevOps workflows. KubeSphere provides developer-friendly wizard web UI, helping enterprises to build out a more robust and feature-rich platform, which includes most common functionalities needed for enterprise Kubernetes strategy, such as the **Kubernetes resource management, DevOps (CI/CD), application lifecycle management, monitoring, logging, Service Mesh (Istio-based), multi-tenancy, alerting and notification, storage and networking, autoscaling, access control, GPU support, etc.**, as well as **multi-cluster management, Network Policy, registry management, more security enhancements** in upcoming releases. KubeSphere is a **distributed operating system with cloud native stack** with Kubernetes as its kernel, and will be very well architected for **plug-and-play integration for third-party applications to boost its ecosystem**.
 
-Not only support upstream Kubernetes, KubeSphere works with its users to alleviate the pain points of Kubernetes including **storage, network, security and ease of use**, also develop new features and capabilities not yet included in upstream Kubernetes. KubeSphere provides a complete user experience around Kubernetes that incorporates set of cloud native ecosystem tools, allows developers and DevOps teams use their favorite tools in a single front-end interface. KubeSphere delivers **consolidated views while integrating a wide breadth of ecosystem tools** upon Kubernetes and offer consistent user experience to reduce complexity. Most importantly, these functionalities are loosely coupled with the platform since they are pluggable and optional based on your demands, it will not impact the flexibilty of Kubernetes.
+KubeSphere teams not only supports upstream Kubernetes, but also works with its users to alleviate the pain points of Kubernetes including **storage, network, security and ease of use**, and develops new features and capabilities that are not yet available in upstream Kubernetes. KubeSphere provides a complete user experience around Kubernetes that incorporates a set of cloud native ecosystem tools, allowing developers and DevOps teams use their favorite tools in a user-friendly console. KubeSphere delivers **consolidated views while integrating a wide breadth of ecosystem tools** around Kubernetes and offers consistent user experience to reduce complexity. Most importantly, these functionalities are loosely coupled with the platform since they are pluggable and optional at your will. Last but not least, it does not change Kubernetes itself anything. In another word, KubeSphere can be deployed on any existing version-compatible Kubernetes cluster.
```

Comment:
`KubeSphere teams not only supports upstream Kubernetes` => `KubeSphere not only supports provisioning upstream Kubernetes cluster`
 
KubeSphere Installer can help users to setup a Kubernetes cluster, this is a feature for new k8s users.